src/bigtiffimage.cpp

```diff
@@ -371,12 +371,12 @@ namespace Exiv2
                                     if ( ::strcmp("Nikon",chars) == 0 )
                                     {
                                         // tag is an embedded tiff
-                                        byte* bytes=new byte[(size_t)count-jump] ;  // allocate memory
-                                        io.read(bytes,(long)  count-jump)        ;  // read
-                                        MemIo memIo(bytes,(long)count-jump)      ;  // create a file
+					std::vector<byte> nikon_bytes(count-jump);
```

Comment:
With c++11 we could use `std::array` ... I want it soon! :stuck_out_tongue:

Reply:
I think `count` is not a constant expression, so `std::array` won't work here.

Reply:
oh right, I did not think about that